features/machine/machine_healthcheck.feature

```diff
@@ -260,3 +260,19 @@ Feature: MachineHealthCheck Test Scenarios
     Then the output should contain:
       | mhc-<%= machine_set.name %>: total targets: 1,  maxUnhealthy: 1%, unhealthy: 1. Short-circuiting remediation |
     """
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-33714
+  @admin
+  Scenario: Leverage OpenAPI validation within MHC
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    Then I use the "openshift-machine-api" project
+
+    # Create MHC with malformed unhealthy nodes value and empty selectors
+    Given I obtain test data file "cloud/mhc/mhc_malformed.yaml"
+    When I run oc create over "mhc_malformed.yaml" replacing paths:
+      | n  | openshift-machine-api |
+    Then the output should contain:
+      |  The MachineHealthCheck "mhc-malformed" is invalid:|
```

Comment:
redundant space after `|`, and missing space after `:`

Reply:
Thanks , updated